README.rst

```diff
@@ -212,3 +234,11 @@ To run the tests, then simply
 .. code-block:: console
 
     RUN_SECURITY=1 RIAK_TEST_HTTP_PORT=18098 python setup.py test
+
+Contributors
```

Comment:
Ah! I'll move those here ... that's what @mjbrender has had us do with the other clients. I know one unit test reads the THANKS file so I'll address that.